aaad/mesos.py

```diff
@@ -51,6 +51,26 @@ def get_tasks(master_url):
     cache[cache_key] = tasks
     return tasks
 
+def get_task_ids_and_resources(master_url):
+    tasks_count = mesos.get_tasks_counts(master_url.rstrip('/'))
+    limit = int(sum(tasks_count.values()))
+    url = '{}/tasks?limit={}'.format(master_url.rstrip('/'), limit)
+    cache_key = 'get_tasks' + master_url
+    tasks = cache.get(cache_key)
+    if tasks:
+        return tasks
+    tasks = {}
+    resp = requests.get('{}'.format(url))
+    if not resp.status_code // 100 == 2:
```

Comment:
Would raising an exception provide more context?
http://docs.python-requests.org/en/master/user/quickstart/#errors-and-exceptions